core/cmd/app.go

```diff
@@ -811,6 +811,12 @@ func NewApp(client *Client) *cli.App {
 					Action: client.Status,
 					Flags:  []cli.Flag{},
 				},
+				{
+					Name:   "render_metrics",
```

Comment:
I just invented this command name, but now I think it would be better to name it `list_metrics`? What do you think?

Reply:
And there is only one other instance of `_` snake case. Everything else is hyphen or camel, so maybe `list-metrics`? Or even flipped as a subcommand, to match others and be forward compatible: `metrics list`.

Also, should it be moved somewhere else? This section is for:
```
Usage:       "Commands for admin actions that must be run locally",
Description: "Commands can only be run from on the same machine as the Chainlink node.",
```